### PR TITLE
Add Dutch translation (nl)

### PR DESCRIPTION
**Pull request by yookoala**
Works done by @meneer. With minor fixes:
  * Squashed.
  * Rebased to current master.
  * Fix incorrect EOL format (change Windows EOL to Unix).

----

Fix #1406.
Fix #1407.
Fix #1408.
Fix #1409.
Fix #1410.

### CODE REVIEW

File: app/assets/javascripts/components/locales/nl.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+const nl = {
+  "column_back_button.label": terug",
```
Comment (Gargron) [inline app/assets/javascripts/components/locales/nl.jsx]: This syntax error is still here

**Review by Gargron — CHANGES_REQUESTED** [review]
This looks much better than the original PRs, thanks! One syntax error inside. Well, at least one. I would always prefer that this stuff is tested before being submitted.

**Comment by roffel** [comment]
Looking good so far. Did not spell check but it seems you forgot `"compose_form.private": "Mark as private",`. 

Keep up the good work! 🇳🇱

File: app/assets/javascripts/components/locales/nl.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  "tabs_bar.compose": "Schrijven",
+  "tabs_bar.home": "Thuis",
+  "tabs_bar.mentions": "Vermeldingen",
+  "tabs_bar.public": "Federatietijdlijn",
+  "tabs_bar.notifications": "Meldingen",
+  "compose_form.placeholder": "Waar ben je mee bezig?",
+  "compose_form.publish": "Toot",
+  "compose_form.sensitive": "Markeer media als gevoelig",
+  "compose_form.spoiler": "Verberg tekst achter waarschuwing",
+  "compose_form.private": "Mark as private",
```
Comment (roffel) [inline app/assets/javascripts/components/locales/nl.jsx]: 🇬🇧 ➡️ 🇳🇱

**Review by roffel — COMMENTED** [review]
Looking good so far. Did not spell check but it seems you forgot `"compose_form.private": "Mark as private",`. 

Keep up the good work! 🇳🇱

File: app/assets/javascripts/components/locales/nl.jsx (hunk 1 above)
Comment (yookoala) [inline app/assets/javascripts/components/locales/nl.jsx]: Fixed in a1276ce

File: app/assets/javascripts/components/locales/nl.jsx (hunk 2 above)
Comment (yookoala) [inline app/assets/javascripts/components/locales/nl.jsx]: Fixed in 0cd6f4d (special thanks to Google Translate) :-)

**Review by Gargron — APPROVED** [review]

**Pull request merged**